packages/near-mobile-wallet/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@near-wallet-selector/near-mobile-wallet",
-  "version": "8.5.3",
+  "version": "8.5.4",
```

Comment:
There's no need to bump the version on this PR, we will do it on the next release.